aerogear/mobile-security-service-operator
# Revert Oauth changes

## Motivation
 

## What
Revert the following changes 
pick 13ad6e2 feat(auth): adding oauth proxy deploy (AEROGEAR-8987)¶
pick 41028c1 fix(oauth): allow users with no namespace access to log in¶

## Why
Unfortunately, after OAuth impl the Service cannot so longer be called by the APP CR. 

## How
- Manually to solve conflicts
- By git rebase remove the above commits and all which is after
- Do a cherry-pick of the others commits.

## Verification Steps
1. Install using this PR + the image `docker.io/cmacedo/mobile-security-service-operator:back`
2. Install all via `make create-all`
3. Check that all is ok, check that the configMap with the env variables of the apps is created and that the app and database are getting these values
4.  Run `oc new-project mobile-security-service-apps`
5. Run `make create-app`
6. Run `make delete-app`

## Checklist:


## Progress


## Additional Notes

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.
